views/view_package.dt

```diff
@@ -98,12 +98,12 @@ block body
 
 		- if( auto pa = "authors" in versionInfo )
 			p#authors.wrap
-				strong Authors: 
```

Comment:
I know, trailing whitespaces is a bad thing, but this one is necessary. It adds spacing after `Authors:`.
Also, space after `Sub packages:`, `Dependencies:` and `System dependencies:` is also required.

That's why it is better to keep coding style fixes and new features in separate PRs.